src/test/java/shopify/createStoreAndInstallSwift.java

```diff
@@ -0,0 +1,406 @@
+package shopify;
+
+import com.github.javafaker.Faker;
+import commons.AbstractPage;
+import commons.AbstractTest;
+import commons.Constants;
+import commons.PageGeneratorManager;
+import org.openqa.selenium.WebDriver;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeTest;
+import org.testng.annotations.Parameters;
+import org.testng.annotations.Test;
+import pageObject.ShopifyPO;
+import pageObject.SwiftPO;
+import pageUI.ShopifyPageUI;
+import pageObject.OberloPO;
+
+import java.io.FileReader;
+import java.io.IOException;
+import java.text.DateFormat;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.Locale;
+import java.util.Random;
+import java.lang.Thread;
+
+public class createStoreAndInstallSwift extends AbstractTest {
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [Nm: Class names should start with an upper case letter (NM_CLASS_NAMING_CONVENTION)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#nm-class-names-should-start-with-an-upper-case-letter-nm-class-naming-convention).
Class names should be nouns, in mixed case with the first letter of each internal word capitalized. Try to keep your class names simple and descriptive. Use whole words-avoid acronyms and abbreviations (unless the abbreviation is much more widely used than the long form, such as URL or HTML).